lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -350,10 +349,33 @@ class CategoryRenderer {
       percentageEl.title = Util.i18n.strings.errorLabel;
     }
 
+    if (this.hasApplicableAudits(category)) {
+      wrapper.classList.add(`lh-gauge__wrapper--${Util.calculateRating(category.score)}`);
+    } else {
+      // render gray n/a for entirely non-applicable categories
+      percentageEl.textContent = 'n/a';
+      percentageEl.style.color = '#757575';
+      percentageEl.title = Util.i18n.strings.notApplicableAuditsGroupTitle;
+    }
+
     this.dom.find('.lh-gauge__label', tmpl).textContent = category.title;
     return tmpl;
   }
 
+  /**
+   * Returns true if an LH category has any non-"notApplicable" audits.
+   * @param {LH.ReportResult.Category} category
+   * @return {Boolean}
+   */
+  hasApplicableAudits(category) {
+    for (const auditRef of category.auditRefs) {
+      if (auditRef.result.scoreDisplayMode !== 'notApplicable') {
+        return true;
+      }
```

Comment:
```suggestion
      if (auditRef.result.scoreDisplayMode !== 'notApplicable') return true;
```

nit

Reply:
> `+      if (auditRef.result.scoreDisplayMode !== 'notApplicable') return true;`
> nit

I'm going to push back on this :) While our loose-ish style and inherited stricter Chromium/Google style guides *allow* single-line conditionals, they never encourage or require them under any circumstances, so I really think it should be up to the author (barring, say, the rest of the file having a very obvious pre-existing style). Personally I'm loath to use them unless the conditional is really short and the braces end up being a visual distraction from the flow.

Buuuut that's mostly moot because I agree with @connorjclark up in https://github.com/GoogleChrome/lighthouse/pull/11024/files#r448083368 that `category.auditRefs.some()` would be perfect for this instead :)

Reply:
> > There was concern that n/a wouldn't be good for internationalization. Were there further discussions I missed? we might wanna go with –
> 
> yeah, I don't know who we approach about this. Maybe we can look through the tc glossary for if this is actually used other places?

Yeah that's a good point; the discussion I was basing the n/a design off of is #9803. @yuinchien do you have any thoughts?